libs/mark-flow-ui/src/components/candidate_contest.tsx

```diff
@@ -351,19 +380,35 @@ export function CandidateContest({
         <WithScrollButtons>
           <ChoicesGrid>
             {orderedCandidates.map((candidate) => {
-              const isChecked = !!findCandidateById(vote, candidate.id);
-              const isDisabled = hasReachedMaxSelections && !isChecked;
+              const isChecked = !!findCandidateInVote(vote, candidate);
+              // In the case of a cross-endorsed candidate, we consider any
+              // version of that candidate as equivalent in tabulation and the voter
+              // may select multiple versions without it impacting the number of selections / overvote trigger.
+              const isEquivalentToSelected = findCandidateInVoteWithAnyParty(
+                vote,
+                candidate.id
+              );
+              const isDisabled =
+                hasReachedMaxSelections &&
+                !isChecked &&
+                !isEquivalentToSelected;
+
               function handleDisabledClick() {
                 handleChangeVoteAlert(candidate);
               }
               let prefixAudioText: ReactNode = null;
               let suffixAudioText: ReactNode = null;
 
-              const numVotesRemaining = contest.seats - vote.length;
+              // Recalculate unique candidates after this selection/deselection
```

Comment:
This comment doesn't quite make sense to me since we're not in a selection/deselection event handler, but simply rendering candidates